libsoftwarecontainer/src/softwarecontainer.cpp

```diff
@@ -61,18 +61,25 @@ SoftwareContainer::SoftwareContainer(const ContainerID id,
     m_config(std::move(config)),
     m_previouslyConfigured(false)
 {
-    checkWorkspace();
+    m_containerRoot = buildPath(m_config->sharedMountsDir(), "SC-" + std::to_string(id));
+    checkContainerRoot(m_containerRoot);
+    if (m_config->enableWriteBuffer()) {
+        tmpfsMount(m_containerRoot, 100485760);
```

Comment:
I'm a bit reluctant on merging things that are super static. Can't we at least have a variable holding the number or something so that it is clear that it shouldn't be static?